BrentOzarULTD/SQL-Server-First-Responder-Kit
# Tweaker

Fixes #1705

Changes proposed in this pull request:
 - See issue for list

How to test this code:
 - Should report correct number of nonclustered indexes in all counts (not disabled or hypothetical)

Has been tested on (remove any that don't apply):
 - Case-sensitive SQL Server instance
 - SQL Server 2016
  - SQL Server 2017
